tests/tests_integration/test_api/test_simulators/test_models.py

```diff
@@ -122,6 +123,24 @@ def test_create_model_and_revisions(
 
         assert model_revision_created is not None
         assert model_revision_created.external_id == model_revision_external_id
+        assert model_revision_created.log_id is not None
```

Comment:
I was thinking (for the future) it would be a nice developer experience, if we could do `model_revision.logs()` or `run.logs()` to retrieve a resource's logs.